app/cdap/components/PipelineTriggersSidebars/index.tsx

```diff
@@ -44,19 +44,24 @@ function PipelineTriggersWithFeatures({
     return null;
   }
 
+  const usePipelineCompositeTriggers = useFeatureFlagDefaultFalse(
+    'pipeline.composite.triggers.enabled'
+  );
+
   return (
     <div className="pipeline-triggers-sidebar-container">
       <PipelineTriggers
         {...{
-          usePipelineCompositeTriggers: useFeatureFlagDefaultFalse(
-            'pipeline.composite.triggers.enabled'
-          ),
+          usePipelineCompositeTriggers,
           pipelineName,
           namespace,
           pipelineType,
         }}
       />
-      <TriggeredPipelines pipelineName={pipelineName} />
+      <TriggeredPipelines
+        pipelineName={pipelineName}
+        usePipelineCompositeTriggers={usePipelineCompositeTriggers}
```

Comment:
don't use the word `use` in react unless you're referring to hooks

https://reactjs.org/docs/hooks-custom.html
`Do I have to name my custom Hooks starting with “use”? Please do. This convention is very important. Without it, we wouldn’t be able to automatically check for violations of [rules of Hooks](https://reactjs.org/docs/hooks-rules.html) because we couldn’t tell if a certain function contains calls to Hooks inside of it.`

Reply:
Thanks! Updated